.azure-pipelines-canary.yml

```diff
@@ -6,7 +6,7 @@ trigger: # Start a new run as soon as a "canaries/build/(...)" branch is created
 resources:
   containers:
     - container: windows
-      image: nventive/vs_build-tools:16.11.9
```

Comment:
I think this PR does too much.
There should be 1 PR to change the pipelines.
There should be **another** PR to switch to file-scoped namespaces.